terraform/files/bin/create_cluster.sh

```diff
@@ -8,6 +8,8 @@ STARTTIME=$(date +%s)
 . ~/.capi-settings
 . ~/bin/cccfg.inc
 
+export PREFIX
```

Comment:
Why do we need that when we have `export PREFIX CLUSTER_NAME` on line n. 30.

Reply:
Looks like I overlooked that. Thanks!

Reply:
Fixed with [8e146ec](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/pull/368/commits/8e146ec4750f76b7cd9bd0a2349c3c48405ae5dc)